__main__.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import os
+from multiprocessing import Pool
+
+from bpm.bpm_extractor import get_song_bpm
+from classification.extractor.high_level_data_extractor import make_high_level_data_file
+from classification.classifier.profile_data_extractor import get_classifier_data
+from database.track_bpm import TrackBPM
+from database.track_party import TrackParty
+from database.track_relaxed import TrackRelaxed
+from database.track_timbre import TrackTimbre
+
+
+def extract_and_save_data_from_song(filename):
+    if not filename.endswith(".wav"):
```

Comment:
I think we should be able to process more than just wav files. DOSE asked if we could do MP3 etc, so i think we should support more file formats...